tests/cross_functional/scale/noobaa/test_warp.py

```diff
@@ -48,52 +48,15 @@ class TestWarp(E2ETest):
             pytest.param(
                 *[1, "OC", None],
             ),
-            pytest.param(
-                *[
-                    1,
-                    "OC",
-                    {
-                        "interface": "OC",
-                        "backingstore_dict": {"aws": [(1, "eu-central-1")]},
-                    },
-                ],
-            ),
-            pytest.param(
-                *[
-                    1,
-                    "OC",
-                    {"interface": "OC", "backingstore_dict": {"azure": [(1, None)]}},
-                ],
-            ),
-            pytest.param(
-                *[
-                    1,
-                    "OC",
-                    {"interface": "OC", "backingstore_dict": {"gcp": [(1, None)]}},
-                ],
-            ),
-            pytest.param(
-                *[
-                    1,
-                    "OC",
-                    {"interface": "OC", "backingstore_dict": {"ibmcos": [(1, None)]}},
-                ],
-            ),
```

Comment:
@udaysk23 Any reason for removing these tests ?

Reply:
We are going with default backingstore based on platform and testing endpoint scales based on IO loads